ui/site/src/tvSingle.ts

```diff
@@ -0,0 +1,22 @@
+import * as xhr from 'common/xhr';
+import throttle from 'common/throttle';
+
+const setZen = throttle(1000, zen =>
+  xhr.text('/pref/zen', {
+    method: 'post',
+    body: xhr.form({ zen: zen ? 1 : 0 }),
+  })
+);
+
+lichess.load.then(() => {
+  lichess.pubsub.on('zen', () => {
+    const zen = $('body').toggleClass('zen').hasClass('zen');
+    window.dispatchEvent(new Event('resize'));
+    setZen(zen);
+  });
+
+  $('body').addClass('playing'); // for zen
+  window.Mousetrap.bind('z', () => lichess.pubsub.emit('zen'));
+  // don't add the click listener to #zentog since one is already
+  // added in boot.ts
+});
```

Comment:
Looks like the lichess tv single game page ui is all in Scala which Im not too familiar with. This pattern of hooking up js to the Scala layouts seems good for this use case but lmk if this will cause unintended issues.

Single tv looks like it already loads in `boot.ts` but that's used in other places where zen mode isnt applicable

Reply:
I don't understand why we need to load this new file, alongside the existing round JS, which already contains all this - in fact that's pretty much raw code duplication.

Why don't we use existing and already loaded round logic instead?